NEWS

```diff
@@ -1,6 +1,11 @@
 libmongoc 2.1.0 (Unreleased)
 ============================
 
+## Fixes
+
+* The uninstall script for Windows is fixed so it no longer "succeeds" with a non-zero `%ERRORLABEL%` value.
+* The uninstall script for non-Windows environments now calls `find "$dir" -maxdepth 1` instead of `ls --almost-all "$dir"` when detecting non-empty directories to fix MacOS compatibility.
```

Comment:
```suggestion
* The uninstall script for non-Windows environments now calls `find "$dir" -maxdepth 1` instead of `ls --almost-all "$dir"` when detecting non-empty directories to fix macOS compatibility.
```

To match casing on https://en.wikipedia.org/wiki/MacOS